crowd_anki/config/config_dialog.py

```diff
@@ -0,0 +1,77 @@
+"""
+Config dialog entry point.
+This module will load the config initially.
+Write actions will be triggered here when the user clicks the dialog "Save All" button.
+"""
+
+from aqt.qt import *
+
+from .config_ui import Ui_Dialog as ConfigUI
+from .config_settings import ConfigSettings
+
+
+class ConfigDialog(QDialog):
+    def __init__(self, config: ConfigSettings,
+                 parent=None):
+        super().__init__(None)
+        self.parent = parent
+        self.config = config
+        self.form = ConfigUI()
+        self.form.setupUi(self)
+        self.ui_initial_setup()
+
+    def accept(self):
+        invalid_values = self.config.find_invalid_config_values()
+        if invalid_values:
+            error_message_popup = QMessageBox(self)
+            message_text = "\" , \"".join(invalid_values)
```

Comment:
nit: you can combine single and double quotes to make this cleaner (i.e. double quotes can be placed in single one or other way around without escaping)